azext_edge/edge/providers/orchestration/resources/instances.py

```diff
@@ -375,50 +390,56 @@ def disable_secretsync(
         confirm_yes: Optional[bool] = None,
         **kwargs,
     ):
-        # TODO: add unit test
         should_bail = not should_continue_prompt(confirm_yes=confirm_yes)
         if should_bail:
             return
 
-        with console.status("Working..."):
-            instance = self.show(name=name, resource_group_name=resource_group_name)
-            resource_map = self.get_resource_map(instance)
-            cl_resources = resource_map.connected_cluster.get_aio_resources(
-                custom_location_id=instance["extendedLocation"]["name"]
-            )
-            secretsync_spcs = self.find_existing_resources(cl_resources=cl_resources, resource_type=SPC_RESOURCE_TYPE)
-            secretsyncs = self.find_existing_resources(
-                cl_resources=cl_resources, resource_type=SECRET_SYNC_RESOURCE_TYPE
+        instance: dict = self.show(name=name, resource_group_name=resource_group_name)
+        # remove the default secret provider class reference
+        default_spc_ref = instance["properties"].pop("defaultSecretProviderClassRef", None)
```

Comment:
does it means in 2507 the default spc resource stored differently place then other spc resources? like below we are getting spc and ssc resources using `get_cl_resources_by_type`, will that included the default spc resource as well? thanks!

Reply:
oh after looking again, is this `instance["properties"]["defaultSecretProviderClassRef"]` just a place to store the ref of the default spc, but to actually delete resource we would still need to call `ssc_mgmt_client.azure_key_vault_secret_provider_classes.begin_delete`?